src/main/java/calculator/InputChecking.java

```diff
@@ -0,0 +1,18 @@
+package calculator;
+
+public class InputChecking {
+
+	public static boolean isBlank(String value)
+	{
+		if(value == null|| value.isEmpty())
+		{
+			return true;
+		}
+		return false;
```

Comment:
이 식은 다음과 같이 간소화 될 수 있습니다!
```suggestion
		return value == null || value.isEmpty();
```

Reply:
제가 너무 장황하게 썼네욥..!! 감사합니다. 수정했습니다.